domain/src/main/java/org/jboss/set/mjolnir/archive/domain/RepositoryFork.java

```diff
@@ -43,6 +35,11 @@
 
     @CreationTimestamp
     private Timestamp created;
+    
+    private Timestamp deleted;
+
+    @Column(name = "user_status")
+    private String userStatus;
```

Comment:
I don't think we need this field. Appears to be unused.

Reply:
Modified these changes